src/impl_aarch64_macos.c

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 Google LLC
```

Comment:
rename `impl_aarch64_macos.c` to  `impl_aarch64_macos_or_iphone.c` like `impl_aarch64_linux_or_android.c`